src/index.ts

```diff
@@ -5,4 +5,4 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-export = {};
+export default {};
```

Comment:
is this index file still required in esm packages?

Reply:
Yes, it's the entry point for oclif/core to figure out the location of the plugin's root directory